app/react/Markdown/components/specs/__snapshots__/MarkdownMedia.spec.js.snap

```diff
@@ -18,6 +18,8 @@ exports[`MarkdownMedia render should render an iframe with the correct video id
           },
```

Comment:
@grafitto this snapshot test is actually failing locally, also on ci, make sure you upgrade node packages and launch tests again.